src/transformers/integrations.py

```diff
@@ -337,6 +341,74 @@ def run_hp_search_sigopt(trainer, n_trials: int, direction: str, **kwargs) -> Be
     return best_run
 
 
+def run_hp_search_wandb(trainer, n_trials: int, direction: str, **kwargs) -> BestRun:
+    from .integrations import is_wandb_available
+
+    assert is_wandb_available(), "This function needs wandb installed: `pip " "install wandb`"
```

Comment:
Please use an if and raise an `ImportError` here, we don't accept new `assert` in the code base :-)

Reply:
Makes sense. I'll make the change.